app/views.py

```diff
@@ -20,6 +21,20 @@ def show_user_input():
     return valid_query_message if content and sms_from and sms_to else (invalid_query_message, invalid_query_status_code)
 
 
+@url.route('/ask_weather')
```

Comment:
@admiral96 
Ideally, URLs endpoints should be *nouns* not *verbs*. Also, looking at the code, this endpoint is returning a *weather forecast* not *weather*.